worldengine/cli/main.py

```diff
@@ -407,12 +407,19 @@ def main():
 
     generation_operation = (operation == 'world') or (operation == 'plates')
 
-    produce_grayscale_heightmap = args.grayscale_heightmap
-    if produce_grayscale_heightmap and not generation_operation:
+    if args.grayscale_heightmap and args.grayscale_heightmap.rfind('.') == -1:
+        print("Invalid filename `{}`, you must specify a valid image extension.".format(
+            args.grayscale_heightmap))
+        sys.exit(-1)
```

Comment:
Should we start using meaningful error codes? Or perhaps adding just a constant for GENERIC_EXIT_CODE?
Just thinking, probably not the useful